app/next-client-app/types/dataset.ts

```diff
@@ -21,3 +14,16 @@ interface DataPartner {
   created_at: string;
   updated_at: string;
 }
+
+interface DataSetSRList {
```

Comment:
Just trying to get my head around this - why is this needed on top of `DataSet`? In `api/datasets.ts` they both are used as return types, but looking at the API, they both return this `DataSetSRList` object (just one in a list, one on it's own).

Reply:
Actually I understand why! Forgot the dataset list endpoint isn't actually useful.